src/ServicePulse.Host/vue/src/components/configuration/HealthCheckNotifications_ConfigureEmail.vue

```diff
@@ -32,12 +32,20 @@ const fromIsValid = computed(() => {
   return from.value && emailRe.test(from.value) ? true : false;
 });
 const toIsValid = computed(() => {
-  return to.value && emailRe.test(to.value) ? true : false;
+  return to.value && validateMultipleEmailsCommaSeparated(to.value) ? true : false;
 });
 const formIsValid = computed(() => {
   return smtpServerIsValid.value && smtpPortIsValid.value && fromIsValid.value && toIsValid.value;
 });
 
+function validateMultipleEmailsCommaSeparated(value) {
+  var result = value.split(",");
+  for (var i = 0; i < result.length; i++)
+  if (!emailRe.test(result[i]))
+     return false;
+  return true;
```

Comment:
This section is hard to read because there aren't braces, and the indentation isn't correct (the `if` line should be indented)

An alternative is to remove the for loop completely.

```suggestion
  return result.every(address => emailRe.test(address));
```